enterprise/cmd/executor/internal/run/setup_ip_tables_windows.go

```
@@ -0,0 +1,7 @@
+package run
+
+import "github.com/sourcegraph/sourcegraph/enterprise/cmd/executor/internal/util"
+
+func SetupIPTables(runner util.CmdRunner, recreateChain bool) error {
+	return nil
```

Rather than returning `nil`, I think it would be better to `panic` here. My understanding is this should never actually be called, and if it is, then we're probably doing something wrong and we should be loud about it.